src/System.Memory/src/System.Memory.csproj

```diff
@@ -20,11 +20,11 @@
     <Compile Include="System\Buffers\ReadOnlySequenceDebugView.cs" />
     <Compile Include="System\Buffers\ReadOnlySequenceSegment.cs" />
     <Compile Include="System\Buffers\ReadOnlySequence_helpers.cs" />
-    <Compile Include="System\Buffers\Binary\Reader.cs" Condition="'$(TargetsAOT)'=='true'" />
-    <Compile Include="System\Buffers\Binary\ReaderBigEndian.cs" Condition="'$(TargetsAOT)'=='true'" />
-    <Compile Include="System\Buffers\Binary\ReaderLittleEndian.cs" Condition="'$(TargetsAOT)'=='true'" />
-    <Compile Include="System\Buffers\Binary\WriterBigEndian.cs" Condition="'$(TargetsAOT)'=='true'" />
-    <Compile Include="System\Buffers\Binary\WriterLittleEndian.cs" Condition="'$(TargetsAOT)'=='true'" />
+    <Compile Include="$(CommonPath)\CoreLib\System\Buffers\Binary\Reader.cs" Condition="'$(TargetsAOT)'=='true'" />
```

Comment:
Until the move propagates to ProjectN, you will need to be careful to not break the corefx build with your intrinsic changes in these files now that they are shared.